odoo/fields.py

```diff
@@ -1451,6 +1456,7 @@ def _setup_attrs(self, model, name):
         # Translated sanitized html fields must use html_translate or a callable.
         if self.translate is True and self.sanitize:
             self.translate = html_translate
+            self.prefetch = False
```

Comment:
I'd say this is redundant.

You can just put the call to `super()` last and let it apply this effect, right?

Reply:
@Yajo the call to `super()` sets the value of all attributes of `self``, so `self.translate` simply crashes if you put the call to `super()` last.

Reply:
I see.

Then we have a similar problem to https://github.com/odoo/odoo/pull/37967/files#r331449030. If some module defines a field like this:

```python
myfield = fields.Html(
  translate=True,
  prefetch=True,
)
```

... it would end up being `prefetch=False`, unexpectedly.

I think then that this should be handled in `__init__()` also. Something like:

```python
def __init__(self, string=Default, **kwargs):
    if kwargs.get("translate", False) and kwargs.get("sanitize", True):
        kwargs["translate"] = html_translate
        kwargs.setdefault("prefetch", False)
```

... and then remove from this class the `_setup_attrs()` method.

That would make the framework behavior more natural and predictable.